maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -379,24 +383,35 @@ class IOSDriver(
     override fun backPress() {}
 
     override fun hideKeyboard() {
-        dismissKeyboardIntroduction()
-        if (isKeyboardHidden()) return
+        val future = executor.submit {
+            dismissKeyboardIntroduction()
 
-        swipe(
-            start = Point(widthPercentToPoint(0.5), heightPercentToPoint(0.5)),
-            end = Point(widthPercentToPoint(0.5), heightPercentToPoint(0.47)),
-            durationMs = 50,
-        )
+            if (isKeyboardHidden()) return@submit
 
-        if (isKeyboardHidden()) {
-            return
-        }
+            swipe(
+                start = Point(widthPercentToPoint(0.5), heightPercentToPoint(0.5)),
+                end = Point(widthPercentToPoint(0.5), heightPercentToPoint(0.47)),
+                durationMs = 50,
+            )
 
-        swipe(
-            start = Point(widthPercentToPoint(0.5), heightPercentToPoint(0.5)),
-            end = Point(widthPercentToPoint(0.47), heightPercentToPoint(0.5)),
-            durationMs = 50,
-        )
+            if (isKeyboardHidden()) {
```

Comment:
super nit: to keep consistency, let's make it the same as in line 389, as a one-line if like `if (isKeyboardHidden()) return@submit`

Reply:
Done ✅